src/AppComplianceAutomation/AppComplianceAutomation.Autorest/custom/Utils.ps1

```diff
@@ -0,0 +1,145 @@
+function Get-ResourceId-Array {
```

Comment:
Could you please add the liscence section to this file as well? And it is recommeded to put this `Utils.ps1` file into the `utils` folder.

Reply:
Added license section.
However, the files under `utils` folder will be removed when I use autorest to generate code, is there anyway to avoid this?

Reply:
Confirmed that it's good to leave `Utils.ps1` here.